### ISSUE TITLE
Request to become maintainer of orphaned package: emamux

### ISSUE THREAD

**Issue by Yuki-Inoue** [melpa/melpa#7695]
Currently, the https://github.com/emacsorphanage/emamux package is under orphanage.  Thus, it cannot receive PRs / bugfix / enhancements.

I've made some contribution against this repository, so I assume I can become maintainer. refer: https://github.com/emacsorphanage/emamux/graphs/contributors

Question: is there any procedure for becoming a maintainer on the orphaned package?

**Comment by riscy** [comment]
Thanks for raising this.  Since this package is in the orphanage, I'd say go right ahead and open a PR.  Apologies for the delayed reply.

But I've renamed the issue to cover the question of whether there's a procedure/policy.  This has come up a couple times recently:

- https://github.com/melpa/melpa/pull/7215
- https://github.com/melpa/melpa/pull/7737
- https://github.com/melpa/melpa/pull/7728

It should be possible to take over unmaintained packages, which is the best case scenario, but I think it would be good if we set down some rough guidelines that people can refer to.  I'll dump some of my thoughts here and mention @melpa/maintainers.

- The new maintainer must have experience.   E.g. maintains/contributes to an existing elisp package, has other visible community contributions, or can be vouched for by such a person
- The package has significant outstanding maintenance issues.  A rock solid package with an inactive maintainer is a non-issue.  Checkdoc warnings and deprecation warnings during a byte-compile are not a big deal.  Medium deals are things like namespace violations.  Big deals are things like being incompatible with current versions of dependencies, or other compile-time/runtime _errors_
- An earnest effort was made to contact the original maintainer (unless the package is somewhere like the orphanage).  Not only via GitHub mentions but also raising issues and reaching out with an email when possible
- The possibility of simply being granted write access to the existing repository has been exhausted
- Decisions should lean in favor of the original maintainer.   "I'll be a better maintainer" might well be true but being the original maintainer should carry some weight

**Comment by riscy** [comment]
Related: https://github.com/melpa/melpa/issues/4384

**Comment by alphapapa** [comment]
FWIW, those sound like great guidelines, @riscy.

**Comment by tarsius** [comment]
> Question: is there any procedure for becoming a maintainer on the orphaned package?

@Yuki-Inoue  Opening an issue here is a good first step. Sorry for not replying earlier. To avoid in the future, mention me explicitly.

I/we generally prefer to give a volunteer commit access to the repository in the orphanage instead of transferring it to the users account. That gives us the opportunity to keep an eye on what's happening and to step in if necessary (doesn't happen often but is has happened) or to appoint someone else should the volunteer later disappear too. Repositories can later be transferred once things have settled.

You have written your own packages before, so that is good. I didn't look closely but those packages generally look like the ought to look. One exception: the closing parenthesis on their own lines make me uneasy. You should generally avoid that except for values of variables where you know that additional values are going to be added over time, and other such things.

**Comment by tarsius** [comment]
@riscy Could you please open a new issue to discuss the general approach? This is going to be a mess otherwise.

**Comment by riscy** [comment]
Yes, I'll do that now and set this one back to its old title -- I was thinking of generalizing this issue but maybe that wasn't the best approach.

**Comment by Yuki-Inoue** [comment]
@tarsius @riscy 

So in this case, should I request commit access to the https://github.com/emacsorphanage/emamux repository? If so, should I open the new issue in that repo?

**Comment by tarsius** [comment]
I've send an invite.

**Comment by Yuki-Inoue** [comment]
thanks, I've joined!

**Comment by tarsius** [comment]
So I am just wondering; when do you plan to get started with this? Haven't seen any commits yet.
